examples/isomorphic/app/src/state/mod.rs

```diff
@@ -1,26 +1,30 @@
+use serde::{Deserialize, Serialize};
 use serde_json;
 use std::cell::Cell;
 use std::rc::Rc;
 
 mod msg;
 pub use self::msg::Msg;
 
+#[derive(Serialize, Deserialize)]
+pub struct Contributor {
+    pub login: String,
+    pub html_url: String,
```

Comment:
Maybe leave a comment saying this is their profile.

And an example such as `https://github.com/austinsheep`